README.rst

```diff
@@ -2,6 +2,8 @@
 django-postcode-lookup
 ======================
 
+https://django-postcode-lookup.readthedocs.io/en/latest/
+
```

Comment:
Can you use a status badge from read the docs? Just like https://github.com/labd/commercetools-python-sdk